test/whl-scale-tests/whl-scale-test.md

```diff
@@ -0,0 +1,255 @@
+# Windows Host Logs Scale Test
+
+## Prerequisites
+- Access to an Azure Subscription
+- A Diagnostics PROD Geneva Account where you have permissions to edit Geneva Log Configurations
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+- Latest sky-dev branch

```

Comment:
Should it also be a prereq to run [./scripts/build/windows/install-build-pre-requisites.ps1](https://github.com/microsoft/Docker-Provider/blob/ci_prod/scripts/build/windows/install-build-pre-requisites.ps1)? This will make sure docker and gcc are installed (also .NET and GO, but not sure if those are needed for hostlogs)

Reply:
Might as well use what is already there to improve usability